### PR TITLE
Refactor WebsiteProvider's WrapperPage

### PR DESCRIPTION
**Pull request by warpech**
As proposed by @tomalec in #105, starting from Starcounter 2.3.1.6523 `starcounter-include` is inserted automatically by middleware so there should be no need for WrapperPage in WebsiteProvider.

This PR removes WrapperPage.

@un-tone pls review. There might be some dragons hiding in the closet. All automated tests and my manual tests seem to be ok, though.

As a drawback, CompositionEditor now cannot save changes in the topmost surface. This is dependent on https://github.com/Starcounter/level1/issues/4017

### CODE REVIEW

**Comment by warpech** [comment]
The build is failing on TC. It seems that is some cleanup problem on the TC: https://teamcity.starcounter.org/viewLog.html?buildId=25626&tab=buildResultsDiv&buildTypeId=Apps_Website_Develop

I have asked @joozek78 for help.

**Comment by warpech** [comment]
Ping @joozek78, @alemoi. Can you pls help? The builds in PRs of the Website app are failing for no apparent reason.

**Comment by alemoi** [comment]
I have changed now the Clean policy to Always. Maybe that helps.

**Comment by joozek78** [comment]
@alemoi Where did you change it? For this build configuration? Project? Globally?

**Comment by warpech** [comment]
Thanks! Trying to re-run

**Comment by joozek78** [comment]
BTW you could just re-run with this one build forced to clean

**Comment by alemoi** [comment]
In VCS settings, applied to all projects: 

image

Lets see if it helps.

**Comment by warpech** [comment]
> BTW you could just re-run with this one build forced to clean

I was trying this two days ago, with no help...

@alemoi your change didn't help either, still failing: https://teamcity.starcounter.org/viewType.html?buildTypeId=Apps_Website_Develop&branch_Apps_Website=simpler-provider&tab=buildTypeStatusDiv

**Comment by joozek78** [comment]
@alemoi can you revert the change? since it didn't help, we don't want it to last - it can slow down further builds

**Comment by joozek78** [comment]
This isn't about TC, it fails on my machine too

---
Pozdrawiam,
Marcin Goliński

**Comment by warpech** [comment]
Sorry, I have just noticed too. I guess the problem is on **me**.

**Comment by joozek78** [comment]
Take a look [here](https://github.com/starcounterprefabs/website/blob/9a9134423741371455cacc0008ab646b047b4873/src/WebsiteProvider/ViewModels/SectionPage.json#L5) - it seems this class doesn't exist (maybe renamed?)

**Comment by alemoi** [comment]
Yes, seems the problem is with something else.

**Comment by warpech** [comment]
Seems that a dog ate half of my refactoring. Sorry guys for trouble. I must fix it in my code.

**Comment by warpech** [comment]
Fixed with a force push. @un-tone could you pls review?

File: src/WebsiteProvider/ViewModels/SurfacePage.json.cs.orig [hunk 1]
```diff
@@ -1,21 +1,22 @@
+using Simplified.Ring6;
 using Starcounter;
 
 namespace WebsiteProvider
 {
+<<<<<<< HEAD
```
Comment (joozek78) [inline src/WebsiteProvider/ViewModels/SurfacePage.json.cs.orig]: that doesn't seem good

Comment (warpech) [inline src/WebsiteProvider/ViewModels/SurfacePage.json.cs.orig]: I should stick to SourceTree, not the command line :|

**Pull request merged**